src/flyout.js

```diff
@@ -187,6 +187,30 @@ export class FlyoutElement extends LitElement {
     `;
   }
 
+  _getFlyoutLinkWithFilename = (url) => {
+    // Get the resolver's filename returned by the application (as HTTP header)
+    // and injected by Cloudflare Worker as a meta HTML tag
+    const metaFilename = document.querySelector(
+      "meta[name='readthedocs-resolver-filename']",
+    );
+
+    // Remove trailing slashes from the version's URL and append the
+    // resolver's filename after removing trailing ``index.html``.
+    // Examples:
+    //
+    //   URL: https://docs.readthedocs.io/en/latest/
+    //   Filename: /index.html
+    //   Flyuout URL: https://docs.readthedocs.io/en/latest/
+    //
+    //   URL: https://docs.readthedocs.io/en/stable/
+    //   Filename: /guides/access/index.html
+    //   Flyuout URL: https://docs.readthedocs.io/en/stable/guides/access/
+    return (
+      url.replace(/\/+$/, "") +
+      metaFilename.content.replace(/\/index.html$/, "")
```

Comment:
I think you need to replace /index.html with /, since the slash is included in the replace.

```suggestion
      metaFilename.content.replace(/\/index.html$/, "/")
```

Reply:
Good point!